src/com/sun/ts/tests/servlet/spec/errorpage/SecondServletErrorPage.java

```diff
@@ -60,7 +61,8 @@ public void service(HttpServletRequest req, HttpServletResponse res)
     pw.println("Servlet Name: " + req.getAttribute(SERVLET_NAME));
     pw.println("Request URI: " + req.getAttribute(REQUEST_URI));
     pw.println("Status Code: " + req.getAttribute(STATUS_CODE));
-    pw.println("Exception Type: " + req.getAttribute(EXCEPTION_TYPE));
+    // with the cast we even enforce it's a Class type
+    pw.println("Exception Type: " + ((Class)req.getAttribute(EXCEPTION_TYPE)).getName());
```

Comment:
This triggers a NullPointerException when the request attribute is null causing one TCK test to fail.
I'll include a fix for this in #338.

Reply:
is it supposed to be null?

Reply:
Yes. If an error status code is set without an exception this attribute will be null.